myapp/logic/trade_logic.py

```diff
@@ -0,0 +1,34 @@
+from myapp import stock_api
+from myapp import models
+
+def extract_info(params):
```

Comment:
"params" is a very difficult signature to work with. The next programer will have to work hard to know what is supposed to go in the params.
consider: 
explicit args, ie. number_of_stocks, etc...
a clall Params with any properties you will need to access

Reply:
That is right, fixed it.